wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift

```diff
@@ -347,7 +347,7 @@ final class ZClientViewController: UIViewController {
     /// - Parameter conversation: conversation to open
     func openDetailScreen(for conversation: ZMConversation) {
         let controller = GroupDetailsViewController(conversation: conversation)
-        let navController = controller.wrapInNavigationController()
+        let navController = controller.wrapInNavigationController(setBackgroundColor: true)
```

Comment:
Is there any case that we would set background color false?

Reply:
@johnxnguyen Good question. I was thinking about that while I was making those changes. I couldn't think of any. 

How do you think we should proceed?

Reply:
Maybe it solves this transparent issue one and for all, but maybe it introduces some other bugs. If I remember correctly, it had something to do with iOS versions. It doesn't hurt to try and run the app on different iOS versions to see if everything is fine.

Reply:
I can definitely do that. Can we merge this PR? And I will create a ticket for further investigating this.

Reply:
@johnxnguyen I will create a ticket and further investigate!